homeassistant/components/esphome/__init__.py

```diff
@@ -228,6 +231,26 @@ def async_on_state(state: 'EntityState') -> None:
     return try_connect
 
 
+async def _async_setup_device_registry(hass: HomeAssistantType,
+                                       entry: ConfigEntry,
+                                       device_info: 'DeviceInfo'):
+    """Set up device registry feature for a particular config entry."""
+    sw_version = 'ESPHome v' + device_info.esphome_core_version
```

Comment:
Why prefix it with `ESPHome`  ? We don't have any hard rules for this, but I would prefer if we keep it using dot notation of numbers.

Reply:
For showing off that it's ~~esphomelib~~ ESPHome and not one of those other ESP firmwares 😉 - at least that was the reason for prefixing it for MQTT.

But here it makes less sense since it's in the "ESPHome" integration screen anyway. I'll remove the prefix